bundle-workflow/src/run_integ_test.py

```diff
@@ -95,13 +96,19 @@ def main():
                     work_dir,
                     args.s3_bucket
                 )
-                test_suite.execute()
+                status = test_suite.execute()
+                if status != 0:
+                    failed_components[component.name] = status
             else:
                 logging.info(
                     "Skipping tests for %s, as it is currently not supported"
                     % component.name
                 )
 
+        if failed_components:
+            for component, status in failed_components.items():
+                logging.info(f'Integration Test Failed for {component} with status code {status}')
```

Comment:
Log passes and failures with something more visible, like `PASS:` and `FAIL`:

Use logging.error for FAILs.